conf/report-template.html

```diff
@@ -119,8 +120,16 @@
         width: auto;
         table-layout: fixed;
       }
-      table.dataTable tbody tr th {
-        width: calc(100% / {{ report.keys()|length + 1}} );
+      table.dataTable tbody tr th#result {
+        width: calc(80% / {{ report.keys()|length}} );
```

Comment:
Why 80%?

Reply:
There is no reason for this particular value but when set to 100% the results table fills the whole width which wasn't happening previously. Should we use the full width?